src/plays/color-palettes-generator/components/utils.js

```diff
@@ -0,0 +1,11 @@
+function componentToHex(c) {
```

Comment:
What is `c` here? can we use some descriptive names?

Reply:
File removed